src/main/java/com/iexec/worker/tee/scone/SconeTeeService.java

```diff
@@ -33,7 +34,7 @@
 @Service
 public class SconeTeeService {
 
-    private static final String PRE_COMPUTE_HEAP_SIZE = "3G";
+    private static final String PRE_COMPUTE_HEAP_SIZE = "4G";
```

Comment:
Since the mrenclave and the heap size are coupled, this information should be fetched from the SMS.
Something like
```
GET /precompute/config
{
    "imageUri": "...",
    "heapSize": "4G"
}
```

Reply:
Will be fixed in another PR